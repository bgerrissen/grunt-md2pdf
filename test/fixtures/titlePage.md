# Title page
![cookie monster](img/cms.jpg)