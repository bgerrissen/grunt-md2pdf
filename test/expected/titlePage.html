<html><head><style></style></head><body><div class="title-page"><h1 id="title-page">Title page</h1>
<p><img src="img/cms.jpg" alt="cookie monster">;</p>
</div><h1 id="chapter-1">Chapter 1</h1>
<h2 id="sub-chapter-1-1">Sub Chapter 1.1</h2>
<h2 id="sub-chapter-1-2">Sub Chapter 1.2</h2>
<h2 id="sub-chapter-1-3">Sub Chapter 1.3</h2>
</body></html>